web/app/themes/dp-timber-theme/functions.php

```diff
@@ -11,6 +11,6 @@
 Timber::init();
 
 // Sets the directories (inside your theme) to find .twig files.
-Timber::$dirname = [ 'templates', 'views' ];
+Timber::$dirname = [ 'templates', 'views', 'components' ];
```

Comment:
why do we need components here as we will be just calling our components from WP twig file.

Reply:
Okay, I will remove it

Reply:
I have updated that